doc/graph/fusion_patterns/matmul_fusion_patterns.md

```diff
@@ -35,6 +35,8 @@ nodes are optional.
    - 0 to 4 Binary or Unary operations are supported in the epilogue subgraph.
    - **Select**: If present, must follow binary/unary operations (if present)
      and can only appear once.
+   - **TypeCast**: If present, must follow select operations (if present) and
```

Comment:
Can you please follow https://uxlfoundation.github.io/oneDNN/dev_guide_graph_convolution_fusion_patterns.html to define the typecast subgraph, thanks

Reply:
got it. thanks for your example.